npm-debug.log.3540228334

```diff
@@ -0,0 +1,25 @@
+0 info it worked if it ends with ok
```

Comment:
should this be git ignored?

Reply:
yes

Reply:
Just removed these and added to the .gitignore